metacatalog/ext/standards_export/extension.py

```diff
@@ -122,3 +139,118 @@ def standards_export(cls, entry_or_resultset: Union[Entry, ImmutableResultSet],
 
         # convert to ElementTree and return
         return ET.ElementTree(ET.fromstring(xml))
+
+    @overload
+    def create_standard_metadata(path: str) -> None: ...
+    @overload
+    def create_standard_metadata(path: Literal[None]) -> ElementTree: ...
+    def create_standard_metadata(session: Session, id_or_uuid: Union[int, str], config_dict: dict, path: str = None, template_path: str = './schemas/iso19115/iso19115-2.j2') -> ElementTree | None:
```

Comment:
a relative path in the function signature like this does not work. The path is evaluated as the function is called and therefore relative to the current working directory of the user, not to the file.

You need to specify a path relative to the **file**, ie. on module level:
```python
TEMPLATE_PATH = os.path.abspath(os.path.join(os.path.dirname(__file__), 'schemas', 'iso19115', 'iso19115-2.j2'))
```
or similar.
The same thing should be done for the config dict. By default, this could be empty.
Ie:
```python
from collections import defaultdict

def better_func_name(..., config_dict: dict = defaultdict(lambda: 'NOT SPECIFIED') ...)
```

Reply:
In addition to that, I get another error, when passing in defaultdicts:

![image](https://user-images.githubusercontent.com/2826034/219052131-849b8d22-da8d-4251-9666-cb9310e9abc0.png)

That means, you update the config_dict? 
We should really not rely on the user passing in **exactly** the correct dict all the time. We should go for a potentially empty default dictionary, which the user may or may not overwrite. I am not sure if the necessary information can be given by the user in any circumstance and always apply.
If vital information is missing, it's ok to return XMLs with missing information.

Reply:
We solved the dictionary stuff for now together.  
In this issue, the future plan for the implementation of the default config dict for ISO export is described: #281